mirascope/core/cohere/stream.py

```diff
@@ -39,3 +45,33 @@ def _construct_message_param(
             message=content if content else "",
             tool_calls=tool_calls,
         )
+
+    def construct_call_response(self) -> CohereCallResponse:
+        if self.message_param is None:
+            raise ValueError(  # pragma: no cover
+                "No stream response, check if the stream has been consumed."
+            )
+        usage = ApiMetaBilledUnits(
```

Comment:
can't usage be `None`? If `self.input_tokens` and `self.output_tokens` are both `None` we should set `usage` to `None`

Reply:
done